.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+### Issue details
```

Comment:
Should there be a `### header` at the beginning of the issue? Will it be ugly when the issues are searched or aggregated? Maybe the first section can be header-less?